tests/test_util.py

```diff
@@ -7,6 +7,8 @@
 # cisagov Libraries
 from admiral import util
 
+# TODO: As part of https://github.com/cisagov/admiral/issues/9, test the
+# actual configuration file used to make the connection.
 VALID_CONFIG_FILE = "tests/data/valid_config.yml"
```

Comment:
Did you want to convert this test config file extension to .yaml as well?
```suggestion
VALID_CONFIG_FILE = "tests/data/valid_config.yaml"
```

Reply:
Don't forget to actually rename the file as well.